### DISCUSSION TITLE
Only 1 movie is imported

### DISCUSSION THREAD

**Discussion by asarnow** [nextpyp/.github#3] [Q&A] [answered]
Hi, I'm trying to try out nextPYP in standalone mode on a workstation. The app is running and my config.toml has a standalone section with the available CPUs and the SLURM section is commented.

When I go to import a movie dataset, the job shows all ~1800 files identified but a symlink is only made to the first one in the project folder, and the job never seems to advance or use any CPU. The "check gain reference" step has the clock icon next to it - it is waiting for something I guess.

Do I need to do something else for standalone mode to work? Thanks!

**Comment by abartesaghi** [comment]
You shouldn't need anything else. What type of import block are you using (from Raw Data, Star, PYP or Session) and are these single-particle or tomo movies? If you click on the icon to the right of the `Check gain reference` job (in the `Jobs` panel) it should tell you the reason the job is not running.

**Comment by asarnow** [comment]
They are tomo movies with the raw data block. I created a new job and it actually does link all the movies and display a preview, but doesn't seem to advance from there. The check gain ref. modal says "the job has started."

The job log in shared/log has a number of connection errors (111) all to port localhost:443/pyp, does that indicate a service isn't running?

**Reply by abartesaghi** [reply, marked as answer]
We don't normally use nextpyp in standalone mode, but apparently you need to set `webhost  = 'http://localhost:8080'` in the configuration file for things to work correctly in this mode.

**Comment by asarnow** [comment]
Great, that's working! Thanks!

**Comment by carsonpurnell** [comment]
I believe I am having a similar error. Launched jobs will hang with spinning cogs forever, with no apparent activity of any kind - no generated workflow files or system hardware activity. I read this thread while setting up the installation, and the suggested webhost hasn't prevented it. the web UI displays no logs or errors, but some of the log files are recording info.
I can't supply the logs at the moment, my remote connection is not allowing file transfers. A /shared/log file has extensive error tracebacks for a failed connection - what I can only imagine is the web service failing to connect to the standalone worker.

**Reply by cuchaz** [reply]
It sounds like maybe the job isn't able to connect to the website? Usually that's fixed by setting the `webhost` option in the config file. But if you've already tried
```
[web]
webhost = 'http://localhost:8080'
```
and you're not using a SLURM cluster, then I'd have to get some more information to suggest another possible solution. The details of the error messages in the `/shared/logs/` files would be helpful.

**Reply by carsonpurnell** [reply]
[out-emifbmZKSCDyRQ7d.log](https://github.com/nextpyp/.github/files/13398895/out-emifbmZKSCDyRQ7d.log)
[out-emjb0Z9r9AFLTWAN.log](https://github.com/nextpyp/.github/files/13398896/out-emjb0Z9r9AFLTWAN.log)
[out-emjb7IkoXCaEctfI.log](https://github.com/nextpyp/.github/files/13398897/out-emjb7IkoXCaEctfI.log)
[out-emjbg2feK8Jysdob.log](https://github.com/nextpyp/.github/files/13398898/out-emjbg2feK8Jysdob.log)
[out-emjhmdy1AZGC0zwE.log](https://github.com/nextpyp/.github/files/13398899/out-emjhmdy1AZGC0zwE.log)
[out-emjlRsLAvwcwKT7u.log](https://github.com/nextpyp/.github/files/13398900/out-emjlRsLAvwcwKT7u.log)
Sorry for the delay, remote access was not cooperating. here's the /shared/log/ files. they appear to be mostly redundant and resulting from some of the various times i tried to run jobs.

**Reply by cuchaz** [reply]
Thanks for sending the logs. Looks like they confirm the theory that HTTP and HTTPs traffic are getting mixed up. Maybe double check that the `web.webhost` setting is using `http` and not `https`?

Also, we made a lot of improvements and simplifications to the install process in [the new v0.5.2 release](https://nextpyp.app/files/pyp/0.5.2/docs/install/install-web.html). Maybe you'd have better luck if you started again with the new install process?

**Reply by carsonpurnell** [reply]
the webhost config setting is definitely http:// in the local config, and navigating to http://localhost:8080 works while https:// does not. I couldn't find any other reference to an https address and there doesn't seem to be a different config shadowing the one i edited.

I will try reinstalling with the new version - looks like it's quite a bit simpler.

**Reply by carsonpurnell** [reply]
Latest version is running and appears to be working properly.

**Reply by cuchaz** [reply]
Awesome! Thanks for sticking with it. Your feedback helped us improve the install process a lot.
